### PR TITLE
GameDB: add fixes for Simple Series vol.7 and other games

### LINKED ISSUE #5319: [BUG]: Boxing Champions/Simple 2000 Vol 7 The Boxing - Character Texture Bug
### Describe the Bug

Characters have issues with some textures. One character Legend is chrome but he's completely black

### Reproduction Steps

gs_20220111150427
gs_20220111151754
gs_20220111152246

### Expected Behavior

Screenshots from real hardware
bc1
bc2
bc3

### PCSX2 Revision

1.7.2212

### Operating System

Windows 10 (64bit)

### If Linux - Specify Distro

_No response_

### CPU

i7 10th Gen

### GPU

RTX 2060

### GS Settings

Default settings

### Emulation Settings

Speed hacks off, no game fixes. EE/IOP and VU's are default. Vulkan render with no hacks.

### GS Window Screenshots

_No response_

### Logs & Dumps

_No response_

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

adds fixes for Simple Series vol.7 , Petz - Horsez 2 
### Rationale behind Changes

Fixes #5319
### Suggested Testing Steps

make sure said games are correct / look at CI

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -10644,6 +10644,8 @@ SLES-51716:
 SLES-51717:
   name: "Boxing Champions"
   region: "PAL-E"
+  gameFixes:
+    - XGKickHack # Fixes grpahical issues.
```
Comment (Blackbird88) [inline bin/resources/GameIndex.yaml:10648]: typo

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -20427,6 +20429,8 @@ SLPM-62193:
 SLPM-62197:
   name: "Simple 2000 Series Vol.07 - The Real Fist Fighter"
   region: "NTSC-J"
+  gameFixes:
+    - XGKickHack # Fixes grpahical issues.
```
Comment (Blackbird88) [inline bin/resources/GameIndex.yaml:20433]: ditto

Comment (Mrlinkwii) [reply bin/resources/GameIndex.yaml]: Taken care of :+1:

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -10644,6 +10644,8 @@ SLES-51716:
 SLES-51717:
   name: "Boxing Champions"
   region: "PAL-E"
+  gameFixes:
+    - XGKickHack # Fixes graphical issues.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:10648]: whitespace at end

File: bin/resources/GameIndex.yaml [hunk 4]
```diff
@@ -20427,6 +20429,8 @@ SLPM-62193:
 SLPM-62197:
   name: "Simple 2000 Series Vol.07 - The Real Fist Fighter"
   region: "NTSC-J"
+  gameFixes:
+    - XGKickHack # Fixes graphical issues.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:20433]: whitespace at end

**Pull request merged**